script/generate-router-examples

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# A script to generate controller examples to STDOUT.
+
+set -e
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 [options]
+
+OPTIONS:
+  -h            Show this message
+  -i IP         The external IP address [default: the IP assigned to `eth0`]
+  -p PORT       The port to start the resource server on [default: 12345]
+USAGE
```

Comment:
I'd rather see a single script that can run any of the example generators instead, of a copied one for each.